app/(auth)/signin/schema.ts

```diff
@@ -0,0 +1,12 @@
+import { zodResolver } from '@hookform/resolvers/zod'
```

Comment:
Please make schema directory and move to there!

Reply:
schema is going to be single, so I just put it here.